src/index.js

```diff
@@ -0,0 +1,70 @@
+import './style.css';
+
+// Array to hold todo list items
+const todoItems = [];
+
+const renderTodo = (todo) => {
+  // select ul element
+  const list = document.querySelector('.todo-list');
+
+  // check if todo is completed
+  const isCompleted = todo.completed ? 'done' : '';
+
+  // create li element
+  const li = document.createElement('li');
+  li.setAttribute('class', `todo-item ${isCompleted}`);
+  li.setAttribute('data-key', todo.index);
+
+  li.innerHTML = `
+    <input id="${todo.index}" type="checkbox"/>
+    <label for="${todo.index}" class="tick js-tick"></label>
+    <span>${todo.description}</span>
+    <i class="fa fa-trash-alt"></i>
+  `;
+
+  list.append(li);
+};
+
+// Function to create new todo object
+const addTodo = (description) => {
+  const todo = {
+    description,
+    completed: false,
+    index: Date.now(),
```

Comment:
- [ ] As per the project requirements stated below, you can see it says that the index should be a number. In your code, you have given the date as the value for the index. kindly fix it according to the requirements.

<img width="865" alt="Screenshot 2022-06-06 at 3 40 14 PM" src="https://user-images.githubusercontent.com/70269507/172162542-dc6398d5-764e-402f-8dc0-26c5e8b28fbe.png">

Reply:
Hey @sja-thedude, 
Thanks for the review. Just changed that.